code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -1,3 +1,5 @@
+#define SPEAK(message) radio.talk_into(src, message, radio_channel, get_spans())

```

Comment:
This just replaces the define with the value it's assigned during preprocessor, you would need to go the same; you've missed out the last two arguments.
Ideally the cloning pod also wouldn't do this as there's little value in defines like this.